atlp-rwanda/Team-Sostene-E-commerce-bn
# #184651060 Login Functionality with Email and Password

#### What does this PR do?
- Login Route and Protected routes
- Deleted Unnecessary index.js
- Also changed the PR template 
- Tested and mocked login with email and password
- Added needed dependencies in readme
#### Description of Task to be completed?
- Login functionality with email and password
- Users cannot access the application without logging in without providing an email and password combination.
#### How should this be manually tested?
- use postman with post request to `http://localhost:{PORT}/users/login`
- use `http://localhost:{PORT}/users/protected-route` to check protected route
- content-type *application/json*
- fields are `email` and `password`
#### What are the relevant pivotal tracker/Trello stories?
- #184651060
